Makefile

```diff
@@ -644,19 +644,22 @@ csharp/OpenMatch/Annotations.cs: third_party/
 	$(PROTOC) third_party/protoc-gen-swagger/options/annotations.proto \
 		-I $(REPOSITORY_ROOT) -I $(PROTOC_INCLUDES) \
 		--plugin=protoc-gen-grpc=grpc_csharp_plugin \
-		--csharp_out=$(REPOSITORY_ROOT)/csharp/OpenMatch
+		--csharp_out=$(REPOSITORY_ROOT)/csharp/OpenMatch \
+		--grpc_out=$(REPOSITORY_ROOT)/csharp/OpenMatch/
 
 csharp/OpenMatch/Openapiv2.cs: third_party/
 	$(PROTOC) third_party/protoc-gen-swagger/options/openapiv2.proto \
 		-I $(REPOSITORY_ROOT) -I $(PROTOC_INCLUDES) \
 		--plugin=protoc-gen-grpc=grpc_csharp_plugin \
-		--csharp_out=$(REPOSITORY_ROOT)/csharp/OpenMatch/
+		--csharp_out=$(REPOSITORY_ROOT)/csharp/OpenMatch/ \
+		--grpc_out=$(REPOSITORY_ROOT)/csharp/OpenMatch/
```

Comment:
I don't _think_ GRPC bindings are necessary for these first two.

Reply:
Good catch, thanks.